modules/ipi-install-troubleshooting-bootstrap-vm.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+[id="ipi-install-troubleshooting-bootstrap-vm_{context}"]
+
+= Bootstrap VM issues
+
+The OpenShift installer spawns a bootstrap node virtual machine, which handles provisioning the OpenShift cluster nodes.
+
+.Procedure
+
+. About 10 to 15 minutes after triggering the installer, check to ensure the bootstrap VM is operational using the `virsh` command:
++
+[source,bash]
+----
+[kni@provisioner ~]$ sudo virsh list
+----
++
+----
+ Id    Name                           State
+ --------------------------------------------
+ 12    openshift-xf6fq-bootstrap      running
+----
+
+
```

Comment:
Notes are formatted as:

[NOTE]
====
Text for admonition
====

per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions